src/collectors/filestat/filestat.py

```diff
@@ -106,7 +118,9 @@ def get_default_config(self):
             'uid_max': 65536,
             'type_include': None,
             'type_exclude': None,
-            'collect_user_data': False
+            'collect_user_data': False,
+            'use_sudo': False,
+            'sudo_cmd': '/usr/bin/sudo',
```

Comment:
these should be added to the `get_default_config_help` section too

Reply:
Added.